### ISSUE TITLE
Previous and next buttons on date selection popup do not appear in screen readers

### ISSUE THREAD

**Issue by smythp** [Bedework/bedework#138]
In "manage events," the calendar popup (accessed by hitting the calendar icon near "start date") does not have previous and next month buttons that appear on screen readers.
